info.md

```diff
@@ -46,6 +46,7 @@ Configuration variables:
 - `host`: Hostname or IP address of your Resol Deltasol KM2 or DL2/DL3
 - `scan_interval` (Optional): Defines update frequency. Optional and in seconds. Defaults to 300 (5 min),
    minimum value is 60 (1 min).
+- `mode` (Optional): Set api mode ["km2", "dlx"]. Defaults to auto detect.
```

Comment:
Would add “set api mode manually” here. And don’t forget to put the same changes in readme.md aswell